### PR TITLE
ADM-733[docs] feat: creat deploy file guide how to deploy by k8s

### PR DESCRIPTION
**Pull request by BoBoDai**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by weiraneve — APPROVED** [review]

File: docs/src/content/docs/en/devops/how-to-deploy-heartbeat-in-multiple-instances-by-k8s.mdx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+kubectl apply -f k8s-backend-service.yaml
+```
+
+You can use `kubectl get services -n heartbeat` to confirm whether the backend service is created successfully.
+
+```shell
+kubectl get services -n heartbeat
+```
+
+#### 3.2 Deploy backend pods
```
Comment (guzhongren) [inline docs/src/content/docs/en/devops/how-to-deploy-heartbeat-in-multiple-instances-by-k8s.mdx:132]: - Deploy pod first then service
- put `-n heartbeat` after kubectl, keep static input before dynamic part

Comment (guzhongren) [reply docs/src/content/docs/en/devops/how-to-deploy-heartbeat-in-multiple-instances-by-k8s.mdx:132]: @BoBoDai

File: docs/src/content/docs/en/devops/how-to-deploy-heartbeat-in-multiple-instances-by-k8s.mdx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+kubectl apply -f k8s-frontend-deployment.yaml
+```
+
+You can use `kubectl get pods -n heartbeat` to confirm whether the frontend pods is created successfully.
+
+```shell
+kubectl get pods -n heartbeat
+```
+
+## K8s resource recycling
```
Comment (guzhongren) [inline docs/src/content/docs/en/devops/how-to-deploy-heartbeat-in-multiple-instances-by-k8s.mdx:274]: `K8s` to `K8S`

Comment (guzhongren) [reply docs/src/content/docs/en/devops/how-to-deploy-heartbeat-in-multiple-instances-by-k8s.mdx:274]: @BoBoDai

**Review by guzhongren — COMMENTED** [review]
@BoBoDai some comments

**Pull request merged**